apps/web/src/components/forms/notification-form.tsx

```diff
@@ -46,39 +48,56 @@ interface Props {
   onSubmit?: () => void;
 }
 
+function getDefaultProviderData(defaultValues?: Notification) {
+  if (defaultValues?.provider === "email") {
+    return JSON.parse(defaultValues?.data).email;
+  } else if (defaultValues?.provider === "discord") {
+    return JSON.parse(defaultValues?.data).discord;
+  }
```

Comment:
Let's add an empty return in case.

```suggestion
  if (defaultValues?.provider === "email") {
    return JSON.parse(defaultValues?.data).email;
  } else if (defaultValues?.provider === "discord") {
    return JSON.parse(defaultValues?.data).discord;
  }
  return "";
```